pkg/partition/partition.go

```diff
@@ -40,6 +40,9 @@ const (
 
 	// NoOfLogicalBlocksForGPTHeader is the no. of logical blocks for the GPT header.
 	NoOfLogicalBlocksForGPTHeader = 1
+	
+	// openEbsNDMPartitionName is the name meta info for openEBS created partitions.
+	const openEbsNDMPartitionName = "OpenEBS_NDM"
```

Comment:
```suggestion
	OpenEBSNDMPartitionName = "OpenEBS_NDM"
```

I am thinking it would be better to have this exported, since we can use it outside the package also.

Reply:
Sorry for going back and forth on the naming.

Reply:
done